config/routes.rb

```diff
@@ -6,6 +6,9 @@
   resource :dashboard, only: [:index]
   resource :updates, only: [:create]
 
+  get "/assign_nodes", to: "dashboard#unassigned_nodes"
+  post "/assign_nodes", to: "dashboard#assign_nodes"
```

Comment:
Me neither as I mentioned here https://github.com/kubic-project/velum/pull/197#discussion_r122936079. Don't worry, I have a list of stuff here to be refactored/reviewed after 1.0. :wink: